### PR TITLE
Add files to sign

### PR DESCRIPTION
**Pull request by ankitkumarr**

### CODE REVIEW

File: build/FileHelpers.cs [hunk 1]
```diff
@@ -42,6 +42,23 @@ public static void RecursiveCopy(string sourcePath, string destinationPath)
...
+        {
+            var allEntries = new List<string>();
+            foreach (var entry in filesAndDirs)
+            {
+                if (entry.Contains("*"))
+                {
+                    var files = Directory.GetFiles(Path.GetDirectoryName(entry), Path.GetFileName(entry), SearchOption.AllDirectories);
+                    allEntries.AddRange(files);
+                }
+                else {
```
Comment (ahmelsayed) [inline build/FileHelpers.cs]: nit: formatting. Put { on a new line

**Comment by ahmelsayed** [comment]
One thing that can make this more robust is to run sigcheck.exe on the output that you expect to all be signed, and fail the build if there is a file that's not signed. Though probably not very urgent to do.

File: build/FileHelpers.cs (hunk 1 above)
Comment (ankitkumarr) [reply build/FileHelpers.cs]: Oops, my bad

**Comment by ankitkumarr** [comment]
@ahmelsayed, I have create this issue https://github.com/Azure/azure-functions-core-tools/issues/1089 and assigned it to me.

**Pull request merged**